### PR TITLE
Ensure secret key isn't reused from the one in example.env

### PR DESCRIPTION
**Pull request by robotdana**
fixes: #242

### CODE REVIEW

**Review by eoinkelly — APPROVED** [review]

**Review by G-Rath — COMMENTED** [review]
LGTM though I think it would be good to include how to generate a nice secure value or at least a link to the documentation to read more.

File: config/initializers/check_env.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# frozen-string-literal: true
+
+# RAILS_SECRET_KEY_BASE should be set to something other than its value in example.env
+
+if Rails.env.production? && Rails.root.join('example.env').read.include?(ENV.fetch('RAILS_SECRET_KEY_BASE'))
+  raise 'RAILS_SECRET_KEY_BASE is unchanged from example.env. ' \
+    'Generate a new one with `ruby -rsecurerandom -e "puts SecureRandom.hex(64)"`'
```
Comment (eoinkelly) [inline config/initializers/check_env.rb:7]: `bundle exec rails secret` might be a more idiomatic choice here 🐘

Comment (robotdana) [reply config/initializers/check_env.rb:7]: well there you go i learn things

**Comment by eoinkelly** [comment]
LGTM :shipit:

**Pull request merged**